prow/plugins/trigger/pull-request.go

```diff
@@ -132,7 +132,11 @@ func buildAllIfTrusted(c Client, trigger *plugins.Trigger, pr github.PullRequest
 			}
 		}
 		c.Logger.Info("Starting all jobs for updated PR.")
-		return buildAll(c, &pr.PullRequest, pr.GUID, trigger.ElideSkippedContexts)
+		elide := false
```

Comment:
nit: isn't preferred style `var elide bool` when it has the default value?

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style